modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/NullDynamicConfigPolicy.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.runtime;
+
+import org.mule.extension.runtime.ExpirationPolicy;
+
+/**
+ * Null implementation of {@link DynamicConfigPolicy}.
+ * <p/>
+ * This is a singleton class. Instances should be get through the {@link #INSTANCE}
```

Comment:
There's no point in doing it differently. It's the same as with the VoidEvent or the NullPayload